ros_ws/src/simulation/simulation_tools/src/simulation_tools/track.py

```diff
@@ -134,4 +142,7 @@ def get_position(self, total_distance, distance_to_center=0):
     (-6.00, -0.17),
     (2.64, -0.36)))
 
+if world_name == "racetrack_decorated_2_big":
+    POINTS = POINTS * 2.5
```

Comment:
```suggestion
    POINTS *= 2.5
```
This changes the data in place, instead of creating a copy.